model/task/task.go

```diff
@@ -169,10 +169,14 @@ type Task struct {
 	// CanReset indicates that the task has successfully archived and is in a valid state to be reset.
 	CanReset bool `bson:"can_reset,omitempty" json:"can_reset,omitempty"`
 
-	Execution           int    `bson:"execution" json:"execution"`
-	OldTaskId           string `bson:"old_task_id,omitempty" json:"old_task_id,omitempty"`
-	Archived            bool   `bson:"archived,omitempty" json:"archived,omitempty"`
-	RevisionOrderNumber int    `bson:"order,omitempty" json:"order,omitempty"`
+	Execution int    `bson:"execution" json:"execution"`
+	OldTaskId string `bson:"old_task_id,omitempty" json:"old_task_id,omitempty"`
+	Archived  bool   `bson:"archived,omitempty" json:"archived,omitempty"`
+
+	// RevisionOrderNumber for user submitted patches is their count of patches
```

Comment:
Phrasing suggestion - `"RevisionOrderNumber for user-submitted patches is the user's current patch submission count"`.